### PR TITLE
AG-7440 Batch 4

### PR DESCRIPTION
**Pull request by StephenCooper**

### CODE REVIEW

File: packages/ag-grid-community/src/validation/errorMessages/errorText.ts [hunk 1]
```diff
@@ -281,9 +281,37 @@ export const AG_GRID_ERRORS = {
...
+        `${feature} is only available if using 'multiRow' selection mode.` as const,
+    129: ({ feature, rowModel }: { feature: string; rowModel: string }) =>
+        `${feature} is only available if using 'clientSide' or 'serverSide' rowModelType, you are using ${rowModel}.` as const,
+    130: () => 'cannot multi select unless selection mode is "multiRow"' as const,
+    131: () => 'cannot range select while selecting multiple rows' as const,
+    132: () => `cannot multi select unless selection mode is 'multiRow'` as const,
+    133: () => 'iconRenderer should return back a string or a dom object' as const,
+    134: ({ iconName }: { iconName: string }) => `Did not find icon ${iconName}` as const,
+    135: () => `Data type of the new value does not match the cell data type of the column` as const,
+    136: () => '' as const,
```
Comment (StephenCooper) [inline packages/ag-grid-community/src/validation/errorMessages/errorText.ts:307]: These will get filled in with more errors in next pass.

File: packages/ag-grid-community/src/validation/rules/gridOptionsValidations.ts [hunk 2]
```diff
@@ -237,6 +238,55 @@ const GRID_OPTION_VALIDATIONS: () => Validations<GridOptions> = () => ({
...
+    },
+    tooltipHideDelay: {
+        validate: (options) => {
+            if (options.tooltipHideDelay && options.tooltipHideDelay < 0) {
+                return 'tooltipHideDelay should not be lower than 0';
+            }
+            return null;
+        },
+    },
+    sortingOrder: {
```
Comment (StephenCooper) [inline packages/ag-grid-community/src/validation/rules/gridOptionsValidations.ts:267]: Duplication for now. Expect to deprecate sortingOrder on the gridOptions

File: packages/ag-grid-community/src/styling/layoutFeature.ts [hunk 3]
```diff
@@ -32,7 +30,7 @@ export class LayoutFeature extends BeanStub {
     }
 
     private updateLayoutClasses(): void {
-        const domLayout = this.getDomLayout();
+        const domLayout = this.gos.get('domLayout') ?? 'normal';
```
Comment (peterjrreynolds) [inline packages/ag-grid-community/src/styling/layoutFeature.ts]: Can this be in defaults?

File: testing/behavioural/src/version.ts [hunk 4]
```diff
@@ -0,0 +1,2 @@
+// DO NOT UPDATE MANUALLY: Generated from script during build time
+export const VERSION = '32.2.0';
```
Comment (peterjrreynolds) [inline testing/behavioural/src/version.ts:2]: Is the script going to update this?

Comment (StephenCooper) [reply testing/behavioural/src/version.ts:2]: yes. Uses the same script as other parts. I have tested this is updated by the script.

**Pull request merged**